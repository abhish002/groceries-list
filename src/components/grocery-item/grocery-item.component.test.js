import React from 'react';
import { create } from 'react-test-renderer';
import GroceryItem from './grocery-item.component';

describe('grocery item component', () => {
  const item = {
    brand: "Jabron",
    category: "beverage",
    item: "Milk",
    qty: 1,
    type: "Organic 2% Milk"
  }

  it('matches the snapshot', () => {
    const groceryItem = create(<GroceryItem groceryItem={item} />);
    expect(groceryItem.toJSON()).toMatchSnapshot();
  });
});
