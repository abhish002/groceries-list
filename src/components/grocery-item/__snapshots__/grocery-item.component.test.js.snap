// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`grocery item component matches the snapshot 1`] = `
<div
  className="grocery-item"
>
  <div
    className="grocery-item__qty"
  >
    QTY: 1
  </div>
  <div
    className="grocery-item__item-container"
  >
    <div
      className="grocery-item__item-container__item"
    >
      Milk
    </div>
    <div
      className="grocery-item__item-container__type"
    >
      Organic 2% Milk
    </div>
  </div>
  <div
    className="grocery-item__category"
  >
    beverage
  </div>
</div>
`;
